pages/signin.html

```diff
@@ -5,11 +5,53 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.2/font/bootstrap-icons.css">
+    <link rel="stylesheet" href="../style/style.css">
+    <title>Sign In</title>
 </head>
 
 <body>
+    <header>
+        <a class="logo" href="#">my<span>Bank</span></a>
 
+    </header>
+    <main>
+        <div class="mainSvg">
+            <h1> We Keep Customerers <span>Coming Back For More !</span> </h1>
```

Comment:
```suggestion
            <span> We Keep Customerers <h1>Coming Back For More !</h1> </span>
```
I prefer to put the h1 inside the span not the other way